response_operations_ui/views/reporting_units.py

```diff
@@ -24,6 +24,7 @@ def view_reporting_unit(ru_ref):
 
     ru_details['surveys'] = sorted(ru_details['surveys'], key=lambda survey: survey['surveyRef'])
 
+    respondent = {}
```

Comment:
If this is to guard against a missing respondent then we should also use `respondent.get('id')` on the `return` line.